tests/pyfunc/test_scoring_server.py

```diff
@@ -346,6 +422,71 @@ def test_parse_with_schema(pandas_df_with_all_types):
     assert all(df["bad_boolean"] == [True, False, True])
 
 
+def test_parse_tf_serving_input():
```

Comment:
Can we also add a test that the json parser detects and correctly parses pandas dataframe with the 2 supported orient encodings ('records' and 'split')?